spec/lib/github_service/commands/review_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+RSpec.describe GithubService::Commands::Review do
+  subject { described_class.new(issue) }
+  let(:issue) { double(:fq_repo_name => "org/repo") }
+  let(:command_issuer) { "europ" }
```

Comment:
Are you **sure** about sharing your login with the world? :wink:

Reply:
@skateman May I change it to yours ?